client/modules/datafiles/src/projects/modals/MetricsModal.tsx

```diff
@@ -0,0 +1,446 @@
+import React, { useState, useEffect, useMemo } from 'react';
+import { Modal, Popover, Select, Table } from 'antd';
+import { QuestionCircleOutlined } from '@ant-design/icons';
+const { Option } = Select;
+
+interface Data1Type {
```

Comment:
Can these interfaces be given more descriptive names? It looks like they represent data from the DOI endpoint and the events endpoint? same for the `data1` and `data2` variables.